tests/e2e_tests/test_data_compatibility.py

```
@@ -126,6 +129,147 @@ def _check_collections(self, host: str, port: int) -> tuple[bool, str]:
+    def _query_collections(self, host: str, port: int) -> tuple[bool, str]:
+        """Run queries against all collections to verify data is actually accessible.
+
+        Sends one query of each kind per collection: dense, sparse, and multivector
+        search, plus scroll with filters covering every payload index type.
+        """
+        base_url = f"http://{host}:{port}"
+
+        for collection in self.EXPECTED_COLLECTIONS:
+            try:
+                # Dense vector search
+                resp = requests.post(
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)